Sprint-3/reading-list/script.js

```diff
@@ -21,3 +21,50 @@ const books = [
   },
 ];
 
+// 〰️ Function to render the reading list dynamically on the webpage
+function renderReadingList(books) {
+  // 〰️ Select the DOM element with id 'reading-list' where the list of books will be displayed
+  const readingList = document.getElementById("reading-list");
+
+  // 〰️ Iterate over each book in the books array
+  books.forEach((book) => {
+    // 〰️ Create a list item <li> for each book to represent it in the reading list
+    const listItem = document.createElement("li");
+
+    // 〰️ Set the background color of the list item based on whether the book has been read
+    // 〰️ If the book is read, set the background color to green, otherwise red
+    listItem.style.backgroundColor = book.alreadyRead ? "green" : "red";
+
+    // 〰️ Create a div to contain the book's title and author
+    const bookInfo = document.createElement("div");
+
+    // 〰️ Create an <h2> element for the book title
+    const bookTitle = document.createElement("h2");
+
+    // 〰️ Create a <p> element for the book author
+    const bookAuthor = document.createElement("p");
+
+    // 〰️ Set the text content of the book title to include both title and author information
+    bookTitle.innerText = `${book.title} by ${book.author}`;
+
+    // 〰️ Create an <img> element to display the book cover image
+    const bookImage = document.createElement("img");
+    // 〰️ Set the src attribute of the image to the book's cover image URL
```

Comment:
dont forget to add an alt tag for the image!

Reply:
Thank you for pointing that out! As you suggested, I've added the alt tag to the book cover images. 🤗